pytition/petition/templates/petition/petition_detail.html

```diff
@@ -53,6 +53,20 @@
 {% block extracss %}
     <link href="{% static "petition.css" %}" rel="stylesheet" type="text/css">
     <style type="text/css">
+    {% if petition.bgcolor != "#FFFFFF" %}
```

Comment:
Maybe just put fixed yellow-ish color for the "theme color" (progress-bar, eaSubmitButton, .sign etc) and use `petition.bgcolor` only for background color.
In a following PR I propose to add a new "theme_color" property to Petition Model and use it for html element theming. That would also allow you to put your hands in the backend code with a non-very-intrusive addition.

Reply:
Ok : I've already set yellow colors in `petition.css`, this style in the template just overrides it.

I've thought of a new theme color too, so I'll take a look at your PR and certainly suggest some things (HTML integration is my first job in web so I can help :) )

Reply:
Ok!